langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -44,6 +51,13 @@ public isolated function isFinite(float x) returns boolean = @java:Method {
 #
 # Exactly one of isFinite, isInfinite and IsNaN will be true for any float value
 #
+# ```ballerina
+# float:Infinity.isInfinite() ⇒ true
```

Comment:
Shall we make this the second example? Same for isNan. Basically have the generic case as the first example.